server/src/lib/utils/RequestUrlGetter.ts

```diff
@@ -12,14 +12,12 @@ export class RequestUrlGetter {
 
     const proto = GetHeader(req, Constants.HEADER_X_FORWARDED_PROTO);
     const host = GetHeader(req, Constants.HEADER_X_FORWARDED_HOST);
-    const port = GetHeader(req, Constants.HEADER_X_FORWARDED_PORT);
     const uri = GetHeader(req, Constants.HEADER_X_FORWARDED_URI);
 
-    if (!proto || !host || !port) {
```

Comment:
Similar point to @jkaberg, the port ought to always be present, but it's possible that the URI is empty. Checking that it's truthy seems like it could end up with false negatives.

I will admit that I hadn't quite validated everything, so my assumptions may be wrong, but is there a reason for making this change?

Reply:
You're right, I will test that out.

Reply:
Authelia does not need X-Forwarded-Port eventually, the port is included in X-Forwarded-Host, building the url out of x-Forwarded-Port would be redundant.

Log that confirms what I'm saying:
```
debug: date='Thu Apr 11 2019 21:52:11 GMT+0200 (CEST)' method='GET', path='/api/verify' requestId='cc6e2002-df4b-4774-9d24-eee7a708648c' sessionId='zpB5cJqbdABqeKC9fhnUjvi3CnlEG_Sm' ip='192.168.240.1' message='Header x-forwarded-host is set to secure.example.com:8080'